Gemfile

```diff
@@ -8,6 +8,9 @@ gem 'rspec'
 gem 'coveralls', require: false
 
 gem 'psych', platforms: :rbx
+gem 'rack', '~> 1.6.5'
+gem 'tins', '~> 1.6.0'
+gem 'term-ansicolor', '~> 1.3.2'
 
```

Comment:
they are used by the gem Nothing was added new. When not frozen - they are resolved to higher versions by bundler and as a result build failed for earlier ruby versions. See https://travis-ci.org/krisleech/wisper-sidekiq/builds/214653091 for details